media/css/mozorg/internet-health.scss

```diff
@@ -0,0 +1,830 @@
+        display: block;
+        margin-top: 20px;
+        overflow: hidden;
+    }
+
+    .footer-hr:after{
+        display: block;
+        overflow: hidden;
+    }
+
+    .footer-top {
+        display: none;
+    }
+
+    @media #{$mq-phone-wide} {
+        .content {
+            @include background-size(260px 260px);
+            padding-top: 280px;
+        }
+
+        h2 {
+            @include font-size(26px);
+        }
+
+        p,
+        .ctas {
+            @include font-size(18px);
+        }
+    }
+
+    @media #{$mq-tablet} {
+        margin-bottom: 40px;
+
+        .content {
+            @include background-size(310px 310px);
+            background-position: top 30px left 30px;
+            padding: 0;
+            position: relative;
+        }
+
+        .inner-container {
+            width: 320px;
+        }
+
+        &.odd .inner-container {
+            float: right;
+        }
+
+        &.even {
+            .content {
+                background-position: top 30px right 30px;
+            }
+
+            .inner-container {
+                float: left;
+            }
+
+            // align footer decoration to the right
+            .footer-top {
+                margin-left: 50%;
+                width: 100%;
+            }
+        }
+
+        h2 {
+            @include font-size(32px);
+            text-align: left;
+        }
+
+        p,
+        .ctas {
+            @include font-size(20px);
+            max-width: none;
+            text-align: left;
+        }
+
+        .ctas {
+            text-align: center;
+        }
+
+        .footer {
+            margin-top: 0;
+        }
+
+        .footer-top {
+            display: block;
+
+            &:after {
+                width: 50%;
+            }
+        }
+    }
+
+    @media #{$mq-desktop} {
+        margin-bottom: 60px;
+
+        .content {
+            @include background-size(350px 350px);
+            background-position: top 30px left 60px;
+            min-height: 450px;
+        }
+
+        .inner-container {
+            width: 440px;
+        }
+
+        h2 {
+            @include font-size(36px);
+        }
+
+        p,
+        .ctas {
+            @include font-size(24px);
+        }
+
+        &.even .content {
+            background-position: top 30px right 60px;
+        }
+    }
+
+    @media #{$mq-desktop-wide} {
+        .content {
+            width: $width-desktop;
+        }
+    }
+}
+
+html[dir="rtl"] #technologies > section {
+    @media #{$mq-tablet} {
+        h2 {
+            text-align: right;
+        }
+
+        p {
+            text-align: right;
+        }
+    }
+}
+
+#privacy {
+    h2 span {
+        background: $colorPrivacy;
+    }
+
+    .content {
+        background-image: url('/media/img/mozorg/internet-health/privacy.svg');
+    }
+
+    .footer-hr:after {
+        @include font-size(36px);
+        color: $colorPrivacy;
+        content: '//////////////////////////////////////////////////';
```

Comment:
Is it worth adding `speak: none;` for this generated content? I'm not sure if / how screen readers might interpret this, but I can imagine it could be pretty annoying? 😉